connect/api.go

```diff
@@ -10,6 +11,20 @@ const (
 	APIVersion = "v4"
 )
 
+func announceSystem(body []byte) (string, string, error) {
+	resp, err := callHTTP("POST", "/connect/subscriptions/systems", body, nil, authToken)
+	if err != nil {
+		return "", "", err
+	}
+	m := make(map[string]string)
```

Comment:
I think it would look cleaner with small local struct with tags and unmarshal. Something like we have in `parseProductsXML`.

Reply:
done